service_container.rst

```diff
@@ -175,26 +175,25 @@ each time you ask for it.
                     <!-- Default configuration for services in *this* file -->
                     <defaults autowire="true" autoconfigure="true" public="false" />
 
-                    <prototype namespace="App\" resource="../src/*" exclude="../src/{Entity,Migrations,Tests}" />
+                    <prototype namespace="App\" resource="../src/*" exclude="../src/{DependencyInjection,Entity,Migrations,Tests,Kernel.php}" />
                 </services>
             </container>
 
         .. code-block:: php
 
             // config/services.php
-            use Symfony\Component\DependencyInjection\Definition;
+            namespace Symfony\Component\DependencyInjection\Loader\Configurator;
 
-            // To use as default template
-            $definition = new Definition();
+            return function(ContainerConfigurator $configurator) {
+                $container = $configurator->services()
+                    ->defaults()
+                    ->autowire()
+                    ->autoconfigure()
+                    ->private();
```

Comment:
Private should be removed every where on 4.2, this could be done in a dedicated PR too.

Reply:
Is that just because services are now private by default? If so, should we have documentation describing that services are private by default starting in 4.2?

Reply:
This is the case since Symfony 4. 4.2 is just now the lowest 4.x branch maintained.